schema.sql

```diff
@@ -5,69 +5,142 @@ drop view if exists unit_attributes;
 drop view if exists unit_time_series;
 drop table if exists prime_mover_types;
 drop table if exists balancing_topologies;
+drop table if exists supply_technologies;
+drop table if exists storage_technologies;
+drop table if exists transmission_lines;
+drop table if exists demand_requirements;
 drop table if exists areas;
 drop table if exists attributes;
 drop table if exists data_types;
 drop table if exists time_series;
+drop table if exists piecewise_linear;
 drop table if exists transmission_interchange;
+drop table if exists entities;
+drop table if exists linkages;
+drop table if exists reserves;
+drop table if exists operational_data;
 
 -- only generation units
 create table  generation_units (
 	unit_id integer primary key,
 	name text not null unique,
-	prime_mover text not null references prime_mover_types(prime_mover),
+	prime_mover text not null,
+	fuel_type text not null,
```

Comment:
I think we need the prime mover table for this.

Reply:
We do have the prime mover table. This is used for the prime mover fk